README.md

```diff
@@ -200,6 +200,7 @@ Options|Default|Description
 ------|------|------
 \-\-bg-color=&lt;R,G,B&gt;|0.2, 0.2, 0.2|Set the window *background color*.<br>Ignored if *hdri* is set.
 \-\-resolution=&lt;width,height&gt;|1000, 600|Set the *window resolution*.
+\-\-position=&lt;x,y&gt;||Set the *window position*, in pixels, starting from bottom left.
```

Comment:
bottom left of what? The monitor? Also, the position corresponds to which corner of the window?